From 5e9d39e8bc7c81651b666b8850b7632622b46941 Mon Sep 17 00:00:00 2001
From: Jonathan G Rennison <j.g.rennison@gmail.com>
Date: Mon, 21 Mar 2016 20:19:29 +0000
Subject: [PATCH 0464/1053] Change: Include _current_company in crashlog AI
 config line

_current_company is not currently logged anywhere in the crashlog.
_local_company is logged, despite being much less useful than
_current_company.
This change logs _current_company alongside _local_company.
---
 src/crashlog.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/crashlog.cpp b/src/crashlog.cpp
index 5aa707eb5d..ab27837078 100644
--- a/src/crashlog.cpp
+++ b/src/crashlog.cpp
@@ -194,7 +194,7 @@ char *CrashLog::LogConfiguration(char *buffer, const char *last) const
 			FontCache::Get(FS_MONO)->GetFontName()
 	);
 
-	buffer += seprintf(buffer, last, "AI Configuration (local: %i):\n", (int)_local_company);
+	buffer += seprintf(buffer, last, "AI Configuration (local: %i) (current: %i):\n", (int)_local_company, (int)_current_company);
 	const Company *c;
 	FOR_ALL_COMPANIES(c) {
 		if (c->ai_info == NULL) {
-- 
2.55.0

